### PR TITLE
rename db field name to displayName

### PR DESCRIPTION
**Pull request by OSPFNeighbour**
rename db field name to displayName and update everywhere that uses it.
update settings page to show displayName and username

### CODE REVIEW

File: client/src/screens/settings.screen.js [hunk 1]
```diff
@@ -137,14 +154,18 @@ class Settings extends Component {
         <View style={styles.userContainer}>
           <View style={styles.userInner}>
             <Icon name="user" size={50} style={styles.userImage} />
-            <Text style={styles.inputInstructions}>{user.username}</Text>
+            <Text style={styles.inputInstructions}>{user.displayName}</Text>
```
Comment (sdunster) [inline client/src/screens/settings.screen.js]: screenshot?

File: server/src/creators.js [hunk 2]
```diff
@@ -108,7 +108,7 @@ export const getCreators = (models) => {
           id,
           username,
           password: hash,
-          name: name || 'Default',
+          displayName: displayName || username,
```
Comment (sdunster) [inline server/src/creators.js]: 👍

File: server/src/test-data/fixtures.js [hunk 3]
```diff
@@ -38,7 +38,7 @@ export const USERS = [
     id: DEFAULT_USER_ID,
     username: DEFAULT_USERNAME,
     password: 'test',
-    name: 'Paddy Platypus',
+    displayName: 'Paddy Platypus',
```
Comment (sdunster) [inline server/src/test-data/fixtures.js]: you updated this one but not the rest...

Comment (OSPFNeighbour) [reply server/src/test-data/fixtures.js]: Others can default :p fuck em!

**Comment by OSPFNeighbour** [comment]
screenshot 2018-02-06 09 23 39

File: server/src/test-data/fixtures.js (hunk 3 above)
Comment (OSPFNeighbour) [reply server/src/test-data/fixtures.js]: Done.

**Pull request merged**